capability/CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ This is an initial release since the fork.
 <!-- Minor releases. -->
 [0.1.1]: https://github.com/kolyshkin/capability/compare/v0.1.0...v0.1.1
 [0.1.0]: https://github.com/kolyshkin/capability/compare/42c35b4376354fd5...v0.1.0
+[0.2.0]: https://github.com/moby/sys/releases/tag/capability%2Fv0.2.0
```

Comment:
Minor nit, but shouldn't this be added to the top of this list instead of the bottom? (they're otherwise in descending order, which matches the order of their usage above)

Reply:
Right; fixed.